README.md

```diff
@@ -1,4 +1,67 @@
 # sync-music
+[![Build Status](https://travis-ci.org/fossdelhi/sync-music.svg?branch=syncing)](https://travis-ci.org/fossdelhi/sync-music)
+
 A project, to ease backup, add metadata, and sync your music using cloud storage. :)
 
-Disclaimer: A repo on an organisation is still signed by your *personal* gpg key
+This app is build up with Python [3.5, 3.6] and Shell [Bash: 4.3] Script.
+Sync-music creates a backup of your music files (.mp3) on dropbox and you get rid of logging into dropbox account to download your music files. In case you loose your hard-disk or change your laptop, using this app, you may get your songs back to your new device.
+
+### Before Installing
+As this app uses dropbox as the host to upload your songs. You need to sign up on dropbox. And generate your tokens from [dropbox developers console](https://www.dropbox.com/developers/apps).
+
+### Installing
+```
+$ cd ~/ && clone https://github.com/fossdelhi/sync-music/
+$ cd sync-music
+$ pip install -r requirements.txt
+$ bash setup.sh
```

Comment:
Can be shortened for `. ./setup.sh` as we cannot be too sure about the emulator running on a user's platform. It might be `zsh` etc. Also, add proper permissions before running a file as application i.e. `chmod +x setup.sh`